README.md

```diff
@@ -80,7 +80,7 @@ also easy to add pytype to your automated testing; see this
 
 Finally, pytype generates files of inferred type information, located by default
 in `.pytype/pyi`. You can use this information to type-annotate the
-corresponding source file, replacing `module.py` with the file's import path:
+corresponding source file, which replaces `module.py` with the file's import path:
```

Comment:
This changes the meaning; "replacing ..." is meant to be an instruction to the reader, not a description of what the type-annotating does.

I agree that the sentence is clunky - suggestions welcome on how to improve it.

Reply:
Ok I think I see what you're trying to do now. 
Correct me if I'm wrong but you're saying in the instruction below 

`merge-pyi -i module.py .pytype/pyi/module.pyi`

the parameter `module.py` should be replaced with the file's import path.

Reply:
That's correct.

Reply:
So I have two suggestions, and one is by saying 'by replacing' which adds an extra word but makes things a lot more clear what is going on, or just make it known in the command that its suppose to be the users file path similar to how you would document a program's parameters

Reply:
Thanks! I'm not entirely sure about calling the parameters filepath and filename, since the second parameter is supposed to be the path corresponding to the module name (e.g., "foo/bar/baz.pyi" for module "foo.bar.baz"), but this is already clearer than what we had before, so I'm going to go ahead and accept this PR and maybe tweak the language later if I think of a way to improve it :)

FYI the process we use for keeping our Google-internal and external code bases in sync will close this PR rather than merging it, but you'll still be credited as the author in the corresponding commit on master.